TowerDefense/GameHUD.cs

```diff
@@ -11,68 +11,74 @@ namespace TowerDefense
     class GameHUD
     {
         Button startButton;
-        Button upgradeButton;
-        MouseHandler mouse;
-        Viewport viewport;
+        ConstructionCard constructionCard;
         ActionCard actionCard;
+        Action startGame;
 
-        internal GameHUD(Viewport viewport, MouseHandler mouse)
+        internal GameHUD(Action startGame)
         {
-            this.viewport = viewport;
-            this.mouse = mouse;
-            startButton = new Button(new Point(10 + 64, 125), ResourceManager.StartButton, HoveringContext.ButtonStart);
-            upgradeButton = new Button(new Point(viewport.Width - 160, (int)(viewport.Height * .55f)), ResourceManager.UpgradeButton, HoveringContext.ButtonUpgrade);
+            this.startGame = startGame;
+            startButton = new Button(new Point(350 + 64, 36), ResourceManager.StartButton, ButtonType.StartButton);
 
             List<Button> buttonList = new List<Button>();
-            buttonList.Add(new Button(Point.Zero, ResourceManager.GenericTower, HoveringContext.ButtonGenericTower));
-            buttonList.Add(new Button(Point.Zero, ResourceManager.CannonTower, HoveringContext.ButtonCannonTower));
-            buttonList.Add(new Button(Point.Zero, ResourceManager.BatteryTower, HoveringContext.ButtonBatteryTower));
-            buttonList.Add(new Button(Point.Zero, ResourceManager.BlastTower, HoveringContext.ButtonBlastTower));
+            buttonList.Add(new Button(Point.Zero, ResourceManager.Block, ButtonType.NothingButton));
+            buttonList.Add(new Button(Point.Zero, ResourceManager.Block, ButtonType.NothingButton));
+            buttonList.Add(new Button(Point.Zero, ResourceManager.Block, ButtonType.NothingButton));
```

Comment:
All these nothing buttons were just place holders so I made sure my menus laid out correctly if I were to add a variable number of buttons.